client_statement.php

```diff
@@ -0,0 +1,188 @@
+    $client_type = nullable_html_entities($row_client_details['client_type']);
+    $client_website = nullable_html_entities($row_client_details['client_website']);
+    $client_net_terms = intval($row_client_details['client_net_terms']);
+
+    $sql_client_unpaid_invoices = "
+    SELECT
+        invoice_id,
+        invoice_number,
+        invoice_prefix,
+        invoice_date,
+        invoice_due,
+        invoice_amount
+    FROM
+        invoices
+    WHERE
+        invoice_client_id = $client_id
+        AND invoice_status NOT LIKE 'Draft'
+        AND invoice_status NOT LIKE 'Cancelled'
+        AND invoice_status NOT LIKE 'Paid'";
+
+    $result_client_unpaid_invoices = mysqli_query($mysqli, $sql_client_unpaid_invoices);
+
+    $currency_code = getSettingValue($mysqli, "company_currency");
+
+    ?>
+
+    <ol class="breadcrumb d-print-none">
+        <li class="breadcrumb-item">
+            <a href="clients.php">Clients</a>
+        </li>
+        <li class="breadcrumb-item">
+            <a href="client_invoices.php?client_id=<?php echo $client_id; ?>"><?php echo $client_name; ?></a>
+        </li>
+    </ol>
+
+    <div class="card card-dark">
+    <div class="card-header py-2">
+        <h3 class="card-title mt-2"><i class="fas fa-fw fa-balance-scale mr-2"></i>Statement for <?php echo $client_name ?></h3>
+        <div class="card-tools">
+            <button type="button" class="btn btn-primary d-print-none" onclick="window.print();"><i class="fas fa-fw fa-print mr-2"></i>Print</button>
+        </div>
+    </div>
+    <div class="card-body p-0">
+        <div>
+            <div class="table-responsive-sm">
+                <table class="table table-sm">
+                    <!-- Past Due Payments -->
+                    <thead class="text-dark">
+                        <tr>
+                            <th>Invoice Number</th>
+                            <th>Date</th>
+                            <th>Due Date</th>
+                            <th>Amount</th>
+                            <th>Balance</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <?php 
+                            while ($row = mysqli_fetch_assoc($result_client_unpaid_invoices)) {
+                                $invoice_number = intval($row['invoice_number']);
+                                $invoice_id = intval($row['invoice_id']);
+                                $invoice_prefix = nullable_html_entities($row['invoice_prefix']);
+                                $invoice_date = nullable_html_entities($row['invoice_date']);
+                                $invoice_amount = floatval($row['invoice_amount']);
+                                $invoice_amount_formatted = numfmt_format_currency($currency_format, $invoice_amount, $currency_code);
+                                $invoice_url = intval("invoice.php?invoice_id=$invoice_id");
```

Comment:
for this one remove intval() it will turn the whole url into a number invoice_id is already esscaped so its not needed for this line

Reply:
@johnnyq this is now fixed.